Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h

```diff
@@ -820,6 +831,178 @@ centroid(const TriangleMesh& tmesh)
   return centroid(tmesh, CGAL::Polygon_mesh_processing::parameters::all_default());
 }
 
+
+/**
+  * \ingroup measure_grp
+  * given two meshes, separates the faces that are only in one, the faces
+  * that are only in the other one, and the faces that are common to both.
+  * Faces with different orientations will be considered as not shared.
```

Comment:
identifies faces only present in `m1` and `m2` as well as the faces present in both polygon meshes. Two faces are identical if they have the same orientation and the same points.